app/scripts/components/data-catalog/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { datasets } from 'veda/thematics';
+
+import { LayoutProps } from '$components/common/layout-root';
+import PageHero from '$components/common/page-hero';
+import { Fold, FoldHeader, FoldTitle } from '$components/common/fold';
+import { Card, CardList } from '$components/common/card';
+import EmptyHub from '$components/common/empty-hub';
+
+import { PageMainContent } from '$styles/page';
+import { thematicDatasetsPath } from '$utils/routes';
+
+/* eslint-disable-next-line fp/no-mutating-methods */
```

Comment:
I wish there was a way to disable this rule for `sort`, but there isn't :(
Only workaround is pretty ugly: https://github.com/jfmengels/eslint-plugin-fp/blob/master/docs/rules/no-mutating-methods.md#pass

Reply:
Well, sort is mutating and will be problematic if not used properly, so maybe it is better to intentional about it?!
The linter could be clever enough to understand that it is a copy and that it is ok! 😅

Reply:
Yes that would be ideal, even `[...arr].sort()` won't make it shut up :/